### PR TITLE
[Task] Delete Assets Follow up SSE

### LINKED ISSUE #177: [Task] Delete Assets Follow up SSE
Once SSE is in place the delete assets needs a follow up to implement the SSEs

### PR DESCRIPTION
**Pull request by lukmzig**
## Changes in this pull request
Resolves https://github.com/pimcore/studio-backend-bundle/issues/177

### CODE REVIEW

File: src/Asset/EventSubscriber/DeletionSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    public function onStateChanged(JobRunStateChangedEvent $event): void
+    {
+        if (
+            $event->getNewState() === JobRunStates::FINISHED->value &&
+            $event->getJobName() === Jobs::DELETE_ASSETS->value
+        ) {
+            $this->publishService->publish(
+                Events::DELETION_FINISHED->value,
+                []
```
Comment (mattamon) [inline src/Asset/EventSubscriber/DeletionSubscriber.php]: We could make it nullable no?

Comment (lukmzig) [reply src/Asset/EventSubscriber/DeletionSubscriber.php]: publish method? yes we could do so I guess

Comment (mattamon) [reply src/Asset/EventSubscriber/DeletionSubscriber.php]: the data. yeah so we do not need to add the empty array

**Review by mattamon — APPROVED** [review]
image

**Pull request merged**